### PR TITLE
Bump puppycrawl checkstyle version from 6.19 to 7.8.2

### PR DESCRIPTION
**Pull request by Shoothzj**
### Changes
Bump puppycrawl checkstyle version to 7.8.2. Fix violation

### CODE REVIEW

**Comment by hangc0276** [comment]
Why not upgrade to a higher version?  7.8.2 was released in 2017.

**Comment by Shoothzj** [comment]
> Why not upgrade to a higher version?  7.8.2 was released in 2017.

@hangc0276 It changes a lot. I think maybe we can change this periodically

File: bookkeeper-server/src/test/java/org/apache/bookkeeper/test/ZooKeeperCluster.java [hunk 1]
```diff
@@ -59,7 +59,7 @@ public interface ZooKeeperCluster {
 
     void killCluster() throws Exception;
 
-    void sleepCluster(int time, final TimeUnit timeUnit, final CountDownLatch l)
+    void sleepCluster(int time, TimeUnit timeUnit, CountDownLatch l)
```
Comment (dlg99) [inline bookkeeper-server/src/test/java/org/apache/bookkeeper/test/ZooKeeperCluster.java:62]: is it related to puppycrawl version upgrade?

Comment (Shoothzj) [reply bookkeeper-server/src/test/java/org/apache/bookkeeper/test/ZooKeeperCluster.java:62]: yes

**Review by hangc0276 — APPROVED** [review]

**Comment by Shoothzj** [comment]
The CI failures are not related to this PR.

**Pull request merged**